editor/components/page-attributes/parent.js

```diff
@@ -72,6 +72,8 @@ const applyWithAPIDataItems = withAPIData( ( props, { type } ) => {
 	const queryString = stringify( {
 		context: 'edit',
 		per_page: 100,
+		orderby: 'menu_order',
+		order: 'asc',
```

Comment:
Is it possible / does it make sense to add tests for this?

Reply:
>Is it possible / does it make sense to add tests for this?

What would you have in mind to test here? Simply that the generated query string matches expectations? There's enough variations that it could be helpful, though maybe non-blocking.